.travis.yml

```diff
@@ -50,6 +49,17 @@ jobs:
         - kill %1
       after_success:
         - authelia-scripts docker push-image --arch=$ARCH
+        - if [[ $TRAVIS_TAG ]]; then docker create --name authelia-binary clems4ever/authelia:$TRAVIS_TAG-$ARCH && docker cp authelia-binary:/usr/app/authelia ./authelia-linux-$ARCH; fi
```

Comment:
It's not gonna be sufficient, we need to tar.gz the binary along with the compiled frontend until we have a better solution for packaging the frontend.

Reply:
You're absolutely right here, that's fine I'll grab the `public_html` folder and tar it up with the binary and adjust accordingly.